src/main/steps/la-portal/check-your-answers/content.test.ts

```diff
@@ -295,10 +297,10 @@ describe('check-your-answer > content', () => {
     expect((form.submit.text as Function)(generatePageContent({ language: 'en' }))).toBe('Save and continue');
   });
 
-  /* test('should contain save-as-draft button', () => {
+  test('should contain welsh submit button in welsh content', () => {
     const generatedContent = generateContent(commonContent);
     const form = generatedContent.form as FormContent;
-    expect((form.saveAsDraft!.text as Function)(generatePageContent({ language: 'en' }))).toBe('Save as draft');
-  }); */
+    expect((form.submit.text as Function)(generatePageContent({ language: 'cy' }))).toBe('Cadw a pharhau');
+  });
```

Comment:
Hi Austen, as discussed this and the "should contain submit button" test aren't accurate reflections of what happens for the user:
English submit buttons are Continue / Save as draft
Welsh submit buttons are Parhau / Cadw fel drafft

I'm happy that fixing the test that causes these tests to be inaccurate is out of scope of this ticket.  However I think you should update both tests with a brief note linking to the jira ticket to fix this issue, e.g. "//See ADOP-nnnn"

Reply:
Comment resolved.